templates/email_request_was_accepted.html

```diff
@@ -0,0 +1,7 @@
+<p>Hi {{ user_name }},</p>
+<br>
+<p>Your request was accepted! See it on the mentorsip application. </p>
```

Comment:
you could give a bit more detail. So that by the email the user can understand which mentorship request was accepted. For example putting which user accepted (user name) and the notes, to understand in what the use will be mentored, or menteed on.